base/random.jl

```diff
@@ -1224,6 +1227,14 @@ let Floats = Union{Float16,Float32,Float64}
             $randfun(                                dims::Integer...               ) = $randfun(GLOBAL_RNG, Float64, dims...)
         end
     end
+
+    for (T,SQRT_HALF) in ((Float64, 0.7071067811865476),
```

Comment:
Last digit should be a 5:   0.7071067811865475
julia> 1/sqrt(BigFloat(2))
7.071067811865475244008443621048490392848359376884740365883398689953662392310596e-01
julia> 1/sqrt(2)
0.7071067811865475